### PR TITLE
Update to License, Notice and Readme files

### PR DESCRIPTION
**Pull request by TheJuanAndOnly99**
~~- JP Morgan Chase should be on the LICENSE and NOTICE files~~
- FINOS "active" and OpenSSF "passing" badges
- Contributing + License sections added to readme

### CODE REVIEW

**Comment by texodus** [comment]
@TheJuanAndOnly99 Thanks for the PR!

You can't reassign the copyright without the consent of the current copyright holders, e.g. the `regular-table` contributors.  Why do we want to reassign the copyright?

File: README.md [hunk 1]
```diff
@@ -556,3 +558,27 @@ yarn start
...
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/regular-table) 
 -->
+
+## Contributing
+For any questions, bugs or feature requests please open an [issue](https://github.com/finos/regular-table/issues)
+
+To submit a contribution:
+1. Fork it (<https://github.com/finos/regular-table/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](./CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
```
Comment (texodus) [inline README.md:568]: Dead link

File: README.md [hunk 2]
```diff
@@ -556,3 +558,27 @@ yarn start
 ## Stats
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/regular-table) 
 -->
+
```
Comment (timkpaine) [inline README.md:561]: This is already in the [CONTRIBUTING.md](https://github.com/finos/regular-table/blob/master/CONTRIBUTING.md) which has been the convention for programming project for decades.

**Review by texodus — CHANGES_REQUESTED** [review]
Can't reassign copyright - need the consent of the original authors (which I grant but there are over a dozen others).

I really think `README.md` stickers are tacky looking but whatever is required I'll add - but the contributing guidelines need some cleaning (and IMO are pretty superfluous ...).

EDIT nvm I'm wrong, this is not a dead link; but still, Github literally inlines these same instructions next to the "fork" button.

File: README.md [hunk 3]
```diff
@@ -556,3 +558,27 @@ yarn start
...
+4. Commit your changes (`git commit -am 'Add some fooBar'`)
+5. Push to the branch (`git push origin feature/fooBar`)
+6. Create a new Pull Request
+
+_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool (or [EasyCLA](https://community.finos.org/docs/governance/Software-Projects/easycla)). Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
+
+
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+
+## License
```
Comment (timkpaine) [inline README.md:578]: This provides no additional useful information on the README while occupying additional screen space. 

Github already puts that on the screen:
Screenshot 2023-01-01 at 3 16 19 PM

and more information can be gleaned from the LICENSE file which has been the convention for...you guessed it, decades.

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by timkpaine** [comment]
unrelated but this looks bad, "openssf best practices: passing" is information sparse given the pixel footprint it takes up. I don't think this badge is necessary and I think it's distracting. 

Screenshot 2023-01-01 at 3 14 26 PM

File: README.md (hunk 1 above)
Comment (timkpaine) [reply README.md:568]: wffm

Screenshot 2023-01-01 at 3 28 52 PM

**Comment by TheJuanAndOnly99** [comment]
Hi @timkpaine @texodus ! Thanks for your feedback I think you raise great points. I have:
- Updated the PR to revert the copyright to how it was. 
- Removed the contributing section from the readme and updated the license section (What do you think of having an AUTHORS file like in [jupyter_templates](https://github.com/finos/jupyterlab_templates/blob/main/AUTHORS)?)
- Removed the OpenSSF badge. Though we would like it to be mentioned somewhere in the readme that this project achieves the OpenSFF passing badge with a link to the project page on the openSFF best practices website https://bestpractices.coreinfrastructure.org/en/projects/6771 (I'll leave it up to you on where we can put this).

Thank you and please let me know what you think!

File: README.md (hunk 2 above)
Comment (TheJuanAndOnly99) [reply README.md:560]: removed

**Comment by timkpaine** [comment]
> * Removed the contributing section from the readme and updated the license section (What do you think of having an AUTHORS file like in [jupyter_templates](https://github.com/finos/jupyterlab_templates/blob/main/AUTHORS)?)

@maoo and i like the `AUTHORS` file approach, im sure @texodus won't mind.

> Removed the OpenSSF badge. Though we would like it to be mentioned somewhere in the readme that this project achieves the OpenSFF passing badge with a link to the project page on the openSFF best practices website https://bestpractices.coreinfrastructure.org/en/projects/6771 (I'll leave it up to you on where we can put this).

maybe in the [development](https://github.com/finos/regular-table#development) block is ok

**Comment by TheJuanAndOnly99** [comment]
> @maoo and i like the `AUTHORS` file approach, im sure @texodus won't mind. 

I've added a AUTHORS file. Feel free to send me list of authors or modify as needed.

> maybe in the [development](https://github.com/finos/regular-table#development) block is ok

Added a small section in the development block

**Comment by TheJuanAndOnly99** [comment]
Hi @texodus @timkpaine thank you for your collaboration. This is good to go on our end, feel free to merge when ready.

File: README.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 </p>
 
 <p align="center">
+<a href="https://community.finos.org/docs/governance/software-projects/stages/active/"><img alt="OpenSFF badge" src="https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-active.svg"></a>
```
Comment (timkpaine) [inline README.md:6]: alt text is wrong

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (TheJuanAndOnly99) [reply README.md]: Good catch, fixed.

**Pull request merged**